site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -81,11 +110,13 @@ const CreateWorkspacePage: FC = () => {
           defaultOwner={me}
           defaultBuildParameters={defaultBuildParameters}
           error={error}
-          template={template as Template}
+          template={template!}
           versionId={versionId}
-          gitAuth={gitAuth as TemplateVersionGitAuth[]}
+          gitAuth={gitAuth ?? []}
+          gitAuthPollingState={gitAuthPollingState}
+          startPollingGitAuth={startPollingGitAuth}
           permissions={permissions as CreateWSPermissions}
-          parameters={parameters as TemplateVersionParameter[]}
+          parameters={parameters!}
```

Comment:
that's a more complicated change that it might seem. I don't actually know what the types this will have at runtime are, or if there's some weird loading logic. this was just a nice quick change to a type assertion that makes the intent more clear, rather than a complete cast that might do something nasty with a very low chance of throwing a compiler error.